UI/obs-app.cpp

```diff
@@ -2009,6 +2009,17 @@ static int run_program(fstream &logFile, int argc, char *argv[])
 	InstallNSApplicationSubclass();
 #endif
 
+#if defined(__linux__)
+	/* NOTE: Under GNOME Wayland session, Qt will always launch OBS with
+	 * X11 (xcb plarform). OBS need to force Qt by setting QT_QPA_PLATFORM
+	 * environment variable to "wayland". The user still can set
+	 * QT_QPA_PLATFORM or the -platform with an other option. */
```

Comment:
Please allow me to suggest the following comment:

```suggestion
	/* Qt doesn't use the Wayland platform on GNOME, so we have to force
	 * it using the QT_QPA_PLATFORM env var. It's still possible to use other
	 * QPA platforms using this env var, or the -platform command line option. */
```

Reply:
I replace the note by yours but the `NOTE` is a keyword for some Code Editor so I keep it.